internal/gitinterface/blob_test.go

```diff
@@ -0,0 +1,98 @@
+package gitinterface
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+
+	"github.com/adityasaky/gittuf/internal/common"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadBlobAndWriteBlob(t *testing.T) {
+	testFileReadSrc, err := os.Open(path.Join("test-data", "test-file-read"))
```

Comment:
@wlynch any tips on how to handle the setup here and in tree_test.go? Also, I think I should update the chdir across the board to clean up after the test...

Reply:
Never mind, I think I worked it out. :smile: